src/elli_http.erl

```diff
@@ -427,7 +427,8 @@ get_request_(Socket, Buffer, Options, {Mod, Args} = Callback) ->
             exit(normal)
     end.
 
-recv_request(Socket, Buffer, Options, {Mod, Args} = _Callback) ->
+recv_request(Socket, Buffer, Options, {Mod, Args} = Callback) ->
+    ok = check_max_buffer_size(Socket, Buffer, Options, Callback, 414),
```

Comment:
Might be a good idea to make `check_max_size` function a bit more generic and apply it in all three cases (request line, headers, body)

Reply:
The `max_body_size` option might be renamed to `max_request_size` as well.